src/common/ChipCell/ChipCell.js

```diff
@@ -88,6 +88,7 @@ const ChipCell = ({
 
   const handleAddNewChip = useCallback(
     chip => {
+      setShow(false)
```

Comment:
if show is already false? do we need unnecessary re-render?
also please rename show to showHiddenChips